lib/shoryuken/worker/inline_executor.rb

```diff
@@ -20,7 +20,7 @@ def perform_async(worker_class, body, _options = {})
         end
 
         def perform_in(worker_class, _interval, body, options = {})
-          perform_async(worker_class, body, options)
+          worker_class.perform_async(body, options)
```

Comment:
@ocisly I couldn't test the code yet, but just eyeballing it, shouldn't this line call `worker_class.new.perform`  like the `perform_async` is calling? Did you have a chance to test this change? Is it fixing what you've reported in #517?

Reply:
Yes, this is the fix. It's modelled after `DefaultExecutor.perform_in`, which also calls `worker_class.perform_async`, which leads to the following sequence of calls:

0. Shoryuken::InlineExecutor.perform_in
1. Shoryuken::Worker.perform_async
2. Shoryuken.worker_executor.perform_async
3. worker_class.new.perform

If we were to call `perform` on the worker instance as you suggest, then the executor won't be invoked at all, leading to the inconsistency described in #517.

Reply:
I've tested this behaviour manually by creating an executor like the one described in the issue and invoking `perform_in` on the `DefaultWorker` from the examples.